tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py35, py36, py37, py38
+envlist = py35, py36, py37, py38
```

Comment:
I would like to support Python 2 because unfortunately [edX](https://github.com/edx/edx-platform) still works on python 2 and one of the reasons of creation this library was ability to configure edX as LTI 1.3 Tool.
I agree that Flask will be Python 3 only but tox should still run python 2 tests

Reply:
Ok, in this case I will remove typehints from flask adapters